### PR TITLE
Psp 2377

### PR DESCRIPTION
**Pull request by devinleighsmith**
Add ltsa tab, required logic to make request/load data.

### CODE REVIEW

**Review by FuriousLlama — APPROVED** [review]

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaTransferSubForm.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { FieldArray, getIn, useFormikContext } from 'formik';
+import { LtsaOrders, TitleTransferDisposition } from 'interfaces/ltsaModels';
+import * as React from 'react';
+import { withNameSpace } from 'utils/formUtils';
+
+import { SectionFieldWrapper } from '../SectionFieldWrapper';
+import { StyledSectionHeader } from '../SectionStyles';
+
+export interface ILtsaTransferSubFormProps {
+  nameSpace?: string;
```
Comment (FuriousLlama) [inline frontend/src/features/mapSideBar/tabs/ltsa/LtsaTransferSubForm.tsx:11]: can this be undefined?

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { InlineInput } from 'components/common/form/styles';
+import { FieldArray, getIn, useFormikContext } from 'formik';
+import { DescriptionOfLand, LtsaOrders } from 'interfaces/ltsaModels';
+import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { withNameSpace } from 'utils/formUtils';
+
+export interface ILtsaLegalNotationsSubFormProps {
+  nameSpace?: string;
```
Comment (FuriousLlama) [inline frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: This field does not seem to be taking undefined values. Is there a reason for leaving the type like that?

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaLandSubForm.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      {lands.length === 0 ? (
+        'this title has no land'
+      ) : (
+        <>
+          <FieldArray
+            name={withNameSpace(nameSpace, 'descriptionsOfLand')}
+            render={({ name }) => (
+              <React.Fragment key={`land-row-${name}`}>
+                {lands.map((land: DescriptionOfLand, index: number) => {
+                  const innerNameSpace = withNameSpace(nameSpace, `descriptionsOfLand.${index}`);
```
Comment (FuriousLlama) [inline frontend/src/features/mapSideBar/tabs/ltsa/LtsaLandSubForm.tsx:30]: would this cause React to complain about the key not being present?

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaTransferSubForm.tsx (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaTransferSubForm.tsx:11]: yes

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: These are composable forms. If used in a different context they may require a different, or possibly undefined namespace.

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaLandSubForm.tsx (hunk 3 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLandSubForm.tsx:30]: Ill double check

File: frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx (hunk 2 above)
Comment (FuriousLlama) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: But are they meant to be common? given the file location and contents, my guess is that they specific for that. But I could be wrong. Same as the other places where the namespace is passed as possibly undefined

Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: Even if they are not common, the order/hierarchy that they are used on this page could change.

Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: Think of it this way, a subform has no way of knowing what the parent formik context looks like. As a result, the object that this form operates on could be:
1) at the root level
2) at some child level
3) part of an array
4) etc.

Passing in the an optional namespace reflects the above uncertainty.

Comment (FuriousLlama) [reply frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx:9]: I see your point, adding that gives some flexibility about how the component could be used in the future. What I am saying is that I have doubts about its future usage, I've seen components that are not reused at all. I prefer the "code for the current intent, not future one" if there is no clear idea of how that would be used in the future. Anyhow, this is a minor thing.

**Review by asanchezr — APPROVED** [review]

**Pull request merged**